package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "hermione-retry-progressive",
-  "version": "0.0.4",
-  "description": "Hermione plugin for adding extra retries at tests with errors matched by patterns.",
+  "name": "@testplane/retry-progressive",
+  "version": "0.0.1",
```

Comment:
`1.0.0`

Reply:
You want to release all of the packages as 1.0.0? I thought about releasing them as 0.1.0